### DISCUSSION TITLE
Error: Transactions for user #1 are off by 0.000000000004996003197112!

### DISCUSSION THREAD

**Discussion by earthling-slavlius** [firefly-iii/firefly-iii#8255] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

After another import something went wrong. I see now this error in logs.
_[x]  Error: Transactions for user #1 are off by 0.000000000004996003197112!_
Chars are not working now.
Clear cache didn't help.

What else I can do to resolve this issue?

### Debug information

Debug information generated at 2023-12-13 19:57:45 for Firefly III version **v6.0.30**.

<table><thead><tr><th colspan="2">System information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Firefly III</td><td>v6.0.30 / v2.0.11 / 21 (exp. 21)</td></tr><tr><td>PHP version</td><td>8.2.12 / apache2handler / Linux</td></tr><tr><td>BCscale</td><td>12</td></tr><tr><td>Error reporting</td><td>Display: Off, reporting: ALL errors</td></tr><tr><td>Max upload</td><td>67108864 (64 MB)</td></tr><tr><td>Database drivers</td><td><strong>*mysql*</strong>, pgsql, sqlite, </td></tr><tr><td>Docker build</td><td><span>#</span>751, base <span>#</span>40</td></tr></tbody></table><table><thead><tr><th colspan="2">Firefly III information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Timezone</td><td>Europe/Berlin + Europe/Berlin</td></tr><tr><td>App environment</td><td>local, debug: false</td></tr><tr><td>Layout</td><td>v1</td></tr><tr><td>Logging</td><td>info, stack / (empty)</td></tr><tr><td>Cache driver</td><td>file</td></tr><tr><td>Default language and locale</td><td>en_US + equal</td></tr><tr><td>Trusted proxies</td><td></td></tr><tr><td>Login provider & user guard</td><td>eloquent / web</td></tr><tr><td>Login headers</td><td>N/A + N/A</td></tr><tr><td>Stateful domains</td><td></td></tr><tr><td>Last cron job</td><td>2023-12-13 19:56:49 (56 seconds ago)</td></tr><tr><td>Mailer</td><td>log</td></tr></tbody></table><table><thead><tr><th colspan="2">User-specific information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>User</td><td><span>#</span>1 of 1</td></tr><tr><td>User flags</td><td><span title="Has rules">:wrench:</span> <span title="Has subscriptions">:email:</span></td></tr><tr><td>Session start</td><td>2023-12-01 00:00:00</td></tr><tr><td>Session end</td><td>2023-12-31 23:59:59</td></tr><tr><td>View range</td><td>1M</td></tr><tr><td>User language</td><td>en_US</td></tr><tr><td>User locale</td><td>en_US</td></tr><tr><td>Locale(s) supported</td><td>en_US.utf8: :white_check_mark:<br>en_US.UTF-8: :white_check_mark:<br></td></tr><tr><td>User agent</td><td>Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36 Edg/120.0.0.0</td></tr></tbody></table>

**Comment by JC5** [comment, marked as answer]
1. Make a backup of your database.
2. Run the following command: `docker exec -it [container] php artisan firefly-iii:force-decimal-size`

That should fix it.

**Comment by earthling-slavlius** [comment]
Thanks. The error was corrected.
Container was restarted but charts still unavailable.
<img width="767" alt="Screenshot 2023-12-13 220654" src="https://github.com/firefly-iii/firefly-iii/assets/52633018/e40aa689-6e2c-49f9-af7a-9ce69a4a7868">

**Reply by JC5** [reply]
There should be errors in your logs, could you share those?

**Comment by earthling-slavlius** [comment]
After adding APP_URL and TRUSTED_PROXIES=* the problem gone :-)
Thank you!
